src/components/organisms/AppRouter/AppRouter.jsx

```diff
@@ -8,30 +7,23 @@ import {
   Redirect,
 } from "react-router-dom";
 import "firebase/analytics";
-import { setUser } from "actions";
+
 import Profile from "pages/Account/Profile";
 import Questions from "pages/Account/Questions";
 import CodeOfConduct from "pages/Account/CodeOfConduct";
 import Login from "pages/Account/Login";
+import Admin from "pages/Account/Admin";
```

Comment:
I think it would make sense to  completely separate the admin from the app. Either a new `admin` folder in `src`, or in `src/pages`. What do you think?

Reply:
i don't think its necessary for the moment, but if/when the admin grows, we should